rest_framework/utils/model_meta.py

```diff
@@ -7,7 +7,7 @@
 """
 from collections import namedtuple
 
-FieldInfo = namedtuple('FieldResult', [
+FieldInfo = namedtuple('FieldInfo', [
```

Comment:
can we verify the change with test?

Reply:
We could have a test asserting `FieldInfo.__name__ == 'FieldInfo'. Is it what you want?

Reply:
that would be enough